Devscord.DiscordFramework/Services/MessagesService.cs

```diff
@@ -46,16 +46,16 @@ public Task SendMessage(string message, MessageType messageType = MessageType.No
             return Task.CompletedTask;
         }
 
-        public Task SendEmbedMessage(string title, string description, IEnumerable<KeyValuePair<string, string>> values)
+        public async Task SendEmbedMessage(string title, string description, IEnumerable<KeyValuePair<string, string>> values)
         {
             var embeds = this._embedMessageSplittingService.SplitEmbedMessage(title, description, values);
-            return this.SendEmbedSplitMessages(embeds);
+            await this.SendEmbedSplitMessages(embeds);
         }
 
-        public Task SendEmbedMessage(string title, string description, IEnumerable<KeyValuePair<string, Dictionary<string, string>>> values)
+        public async Task SendEmbedMessage(string title, string description, IEnumerable<KeyValuePair<string, Dictionary<string, string>>> values)
         {
             var embeds = this._embedMessageSplittingService.SplitEmbedMessage(title, description, values);
-            return this.SendEmbedSplitMessages(embeds);
+            await this.SendEmbedSplitMessages(embeds);
```

Comment:
pewnie to mi się dodało podczas merga, ale zmień na returny z powrotem

Reply:
nie bo mi wyskakuje błąd kompilacji CS1997

Reply:
w sensie, że pisze, że nie może być po return żadnego wyrażenia czy instrukcji. Musiałoby być Task<T> aby to działało